### PR TITLE
Removes Security and Engineering's Medbay Access

### PR DESCRIPTION
**Pull request by MarinaGryphon**
Because Sec won't stop standing in front of the fucking door on harm intent.

It is not Security's place to barge into Medical unannounced. It is not Security's place to stir shit in Medbay's lobby. It is Security's place to listen to what Medical says while they're in Medical, but it seems people can't understand that when the temptation of walking right in and shouting is there.

### CODE REVIEW

File: html/changelogs/MDP-secgetout.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+author: MoondancerPony
+delete-after: True
+
+changes: 
+  - tweak: "Security no longer has access to Medical."
+  - rscdel: "Killed innocent kittens."
```
Comment (MarinaGryphon) [inline html/changelogs/MDP-secgetout.yml]:
```suggestion
```

File: html/changelogs/MDP-secgetout.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+author: MoondancerPony
+delete-after: True
+
+changes: 
+  - tweak: "Security no longer has access to Medical."
```
Comment (MarinaGryphon) [inline html/changelogs/MDP-secgetout.yml]:
```suggestion
  - maptweak: "Security and Engineering no longer have direct access to Medical and must be allowed in."
```

**Comment by ProbablyCarl** [comment]
It'd be easier to, and much more sensible to change the area access type, and restrict it to Medical.

**Comment by Roland410** [comment]
I think this should be a broader range PR such as removing medical and engineering's access to security as well, considering they have no reason to be able to come and go as they wish. Not sure if other departments are affected or not, but unless major access changes happened since the Bay forking, there will be more I forgot.

**Comment by MarinaGryphon** [comment]
> It'd be easier to, and much more sensible to change the area access type, and restrict it to Medical.

Ìt wasn't set to autoset, so no, it would not be easier. You would also have to remove the mapped in access vars to prevent Travis errors, meaning an extra line change and then some code changes.

**Comment by MarinaGryphon** [comment]
> I think this should be a broader range PR such as removing medical and engineering's access to security as well, considering they have no reason to be able to come and go as they wish. Not sure if other departments are affected or not, but unless major access changes happened since the Bay forking, there will be more I forgot.

Out of scope. This is literally only to prevent Sec from loitering in the medbay hallway and getting patients killed, which is a pretty frequent problem.

**Pull request merged**